projboard/tests/test_client.py

```diff
@@ -111,3 +111,7 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+# def test_read_article_page(client):
+#     response = client.get("/article/")
```

Comment:
Is this section will be updated in the future or this comment can be deleted?
Also, isn't it missing `response.status_code 200` assertion?

Reply:
definitely , but it still draft .

Reply:
Done